### DISCUSSION TITLE
Multiple /dev/videoX in streams.txt

### DISCUSSION THREAD

**Discussion by emasson-maxx** [ultralytics/yolov5#8525] [General]
Hello Yolov5 community,
I need your help on the following requirement. I need to support multiple cameras (up to 4) while only using one inference model for object detection. Based on my research it is possible but the "type" of stream doesn't specify (to be confirmed) the format I need.  

Please help!

So can I define my UVC-cameras via streams.txt as followed?
/dev/video1
/dev/video3
/dev/video5
/dev/video7

**Comment by glenn-jocher** [comment]
@emasson-maxx yes that might work, though usually stream addresses start with rtsp://, rtmp:// or https://. If you put 4 addresses in your file then inference will run at batch-size 4 (one image per stream).

You can test streams individually also, i.e.
```
python detect.py --source rtsp://stream/address/here
```
